### ISSUE TITLE
retraining face detection

### ISSUE THREAD

**Issue by natxopedreira** [opencv/openvino_training_extensions#304]
Hello

I want to retrain a face detection model, incorporating a thermal image database of arround 2k images to use with a thermal camera. For my needs i think is fine-tuning what i need, correct me if im wrong

`Fine-tuning from pre-trained weights. If the dataset is not big enough, then the model tends to overfit quickly, forgetting about the data that was used for pre-training and reducing the generalization ability of the final model. Hence, small starting learning rate and short training schedule are recommended.`

What it means small starting learning rate? and short training schedule? where are those parameter?

Is there any tutorial ??

Im not sure how to proced, maybe can confirm:

`data_root = 'data/WIDERFace/'` here should point to my data folder
`create an annotation in MSCOCO  format on my data`
`total_epochs = 70` in the docs is mentioned to increase.. how much?
`regarding starting learning rate and short training schedule` what parameter to modify

Thank you!!!

**Comment by AlexanderDokuchaev** [comment]
Hi @natxopedreira 
All existed instruction you can find in [readme.md](https://github.com/opencv/openvino_training_extensions/blob/develop/pytorch_toolkit/object_detection/face-detection/readme.md)

Learnin rate set in [config.py](https://github.com/opencv/openvino_training_extensions/blob/develop/pytorch_toolkit/object_detection/face-detection/face-detection-0100/config.py#L126).

To fine-tune our pretraining model you should download [snapshot by link from table](https://github.com/opencv/openvino_training_extensions/blob/develop/pytorch_toolkit/object_detection/face-detection/readme.md). 
After set path of snapshot to `resume_from` field in (config.py)[ 
https://github.com/opencv/openvino_training_extensions/blob/develop/pytorch_toolkit/object_detection/face-detection/face-detection-0100/config.py#L143]. In this case value of learning rate get from snapshot and you should not change it.

>data_root = 'data/WIDERFace/' here should point to my data folder
create an annotation in MSCOCO format on my data

Yes, it's correct.

>total_epochs = 70 in the docs is mentioned to increase.. how much?

Number of epoch depends from datasets, you can start from 75 (it's means that you will train 5 epochs on your dataset).

**Comment by natxopedreira** [comment]
Thank you, it worked... more or less
